testsuite/shared/src/main/java/org/jboss/as/test/integration/management/util/CustomCLIExecutor.java

```diff
@@ -254,8 +255,12 @@ private static String executeProcess(final CliCommandBuilder commandBuilder, fin
         } finally {
             cliProc.destroyForcibly();
         }
-
-        final String cliOutput = out.toString();
+        String cliOutput = null;
+        try {
+            cliOutput = out.toString(StandardCharsets.UTF_8.name());
```

Comment:
On The CLI side, although no interactive session has been started, the CLI will print-out content using the default JVM platform encoding (that can be something different than UTF-8). 
Seems safer to continue to rely on the default encoding.